openbb_terminal/cryptocurrency/due_diligence/dd_controller.py

```diff
@@ -176,8 +176,12 @@ def __init__(
                 "--descend": {},
             }
             choices["events"]["-s"] = {c: None for c in coinpaprika_view.EVENTS_FILTERS}
-            choices["twitter"]["-s"] = {
-                c: None for c in coinpaprika_view.TWEETS_FILTERS
+            choices["twitter"] = {
+                "--sort": {c: {} for c in coinpaprika_view.EX_FILTERS},
```

Comment:
Why the change from `TWEETS_FILTERS` to `EX_FILTERS`?

![image](https://user-images.githubusercontent.com/48914296/199060847-cc132596-49ab-46e0-ad4e-b05aace1c6d8.png)

![image](https://user-images.githubusercontent.com/48914296/199060881-e1308686-d17d-4ae1-8016-40851505f652.png)

Looking at the df seems we want the `TWEETS_FILTERS`:
![image](https://user-images.githubusercontent.com/48914296/199061099-6709363f-8b74-4468-996b-b7d9d7df2492.png)

Reply:
Yessirrr my bad. Thanks for the review